.claude/commands/jira-register.md

```
@@ -0,0 +1,232 @@
+... 반복 ...
+        ↓
+모든 Sub-task 완료
+        ↓
+/jira-status MEMBER-123 done  (Parent Issue 완료)
+```
+
+---
+
+## 에러 처리
+
+### Plan 없음
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)